core/modules/wsched/BlendScheduler.cc

```diff
@@ -67,12 +67,25 @@ BlendScheduler* dbgBlendScheduler=nullptr; ///< A symbol for gdb
 ////////////////////////////////////////////////////////////////////////
 // class BlendScheduler
 ////////////////////////////////////////////////////////////////////////
-BlendScheduler::BlendScheduler(std::shared_ptr<GroupScheduler> group,
-                               std::shared_ptr<ScanScheduler> scan)
-    : _group{group}, _scan{scan}
+BlendScheduler::BlendScheduler(std::string const& name,
+                               int schedMaxThreads,
+                               std::shared_ptr<GroupScheduler> const& group,
+                               std::shared_ptr<ScanScheduler> const& scanFast,
+                               std::shared_ptr<ScanScheduler> const& scanMedium,
+                               std::shared_ptr<ScanScheduler> const& scanSlow)
```

Comment:
Could/should BlendScheduler be more generic, rather than having a fixed collection of scan scheds?